projects/packages/forms/src/contact-form/css/file-field.css

```diff
@@ -134,14 +138,17 @@
 	flex-grow: 2;
 }
 
+.jetpack-form-file-field__file-name {
+	text-align: left;
+	font-weight: bold;
```

Comment:
Wondering if we should instead rely on `strong` tag and let the theme set exact font-weight it needs?

Reply:
Good call.